src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -546,7 +561,11 @@ class RecipeMarkdownGenerator : Runnable {
             changelog.appendText("## New Recipes\n")
 
             for (newRecipe in newRecipes) {
-                changelog.appendText("\n* [${newRecipe.name}](${newRecipe.docLink}): ${newRecipe.description.trim().replace("\n", "<br />")} ")
+                changelog.appendText(
+                    "\n* [${newRecipe.name}](${newRecipe.docLink}): ${
+                        newRecipe.description.trim().replace("\n", "<br />")
+                    } "
+                )
```

Comment:
This and other changes further down came from applying the formatter; figured instead of fighting/reverting it just accept and move on.